add note about host tags format (#3639)
DataDog/datadog-agent: docs/agent/changes.md

### Agent configuration file

> Please refer to [this section][config] of the documentation for a detailed list

Reviewer: ```suggestion
Refer to the [Configuration Options][config] for a detailed list
```
Should this link be pointing to a tree branch?

> of the configuration options that were either changed or deprecated in the new Agent.

Reviewer: ```suggestion
of the options that were either changed or deprecated in the new Agent.
```

    Kubernetes and Openshift (default to true). This will avoid removing
    them from the exclude list by error

> The format of the `DD_TAGS`  global option has changed. In v5 it is comma-separated; in v6 it is space-separated, i.e. `simple-tag-0 tag-key-1:tag-value-1`.

Reviewer 2: I think we should remove this line. It's under the `docker check` section whereas it actually applies to the containerized Agent, which are 2 different things

Author: 👍
